BusScheduleDB/2 data.sql

```diff
@@ -0,0 +1,54 @@
+use BusScheduleDB
+go
+
+delete StudentTransportation
+go
+
+insert StudentTransportation(LastName, FirstName, Class, Address, Stop, AMRoute, PMRoute, FridayRoute)
+select 'Lincoln', 'Abraham', '4-101', '19th ave', 'corner 45th st',	'A-JKL', 'P-JKL', 'F-JKL'
```

Comment:
I see that i gave wrong sample data but I had in mind address of students home and then the stop would be the corner ave and street together.

Reply:
Yes, I was confused about that...but I don't have the student's home addresses so what should I put in?